### DISCUSSION TITLE
invalid run command!

### DISCUSSION THREAD

**Discussion by codewrangler1** [Genio-The-Haiku-IDE/Genio#422] [General]
I was just trying out the IDE and created one of the template projects. Building works and creates an app, but if I try to use the "play" button, I get the error, "invalid run command !"...anyone seen this?

**Comment by jackburton79** [comment]
Hi @codewrangler1 
I think the executable is generated in a different folder than the one specified in the project settings.
Usually when building from makefile, haiku puts the object files and the target executable in a folder like "objects.x86_64.gcc13" (depending on the architecture and the gcc version)
Maybe the run command in project settings doesn't take this into account.
Please post a screenshot of the project settings window, in particular the "build/release" and "build/debug" sections.

**Reply by codewrangler1** [reply]
The screenshots you  asked for...I am running up to date Beta4 and latest Genio from HaikuDepot...

<img width="1680" alt="Screenshot 2024-07-30 at 10 01 41 AM" src="https://github.com/user-attachments/assets/fb52bbfd-22ba-481f-a4b1-0ab486f8174e">
<img width="1680" alt="Screenshot 2024-07-30 at 10 02 31 AM" src="https://github.com/user-attachments/assets/008a67ed-7036-4743-bfcc-c65c6670d31c">
<img width="1680" alt="Screenshot 2024-07-30 at 10 02 37 AM" src="https://github.com/user-attachments/assets/12e04b14-7450-4e0e-8bac-208fad349516">
<img width="1680" alt="Screenshot 2024-07-30 at 10 02 42 AM" src="https://github.com/user-attachments/assets/8235f2c1-96e7-45b4-a554-b516421f3a86">

**Reply by codewrangler1** [reply]
I added 'MyApp' to the target field and it now works.

**Reply by jackburton79** [reply]
Yes, that's what I was going to suggest.
Genio can guess the build command but not the target executable (yet?)

**Reply by codewrangler1** [reply]
That should be an easy fix, as it's right in the makefile.

**Reply by jackburton79** [reply]
We don't parse the makefile (yet), though

**Reply by codewrangler1** [reply]
Gotcha
